tests/cache_file_generator.py

```diff
@@ -0,0 +1,29 @@
+import logging
+import os
+import sys
+import time
+
+from portalocker import exceptions
+
+from msal_extensions import FilePersistence, CrossPlatLock
+
+
+def _acquire_lock_and_write_to_cache(cache_location, sleep_interval):
```

Comment:
I believe @bgavrilMS wants:

> accepts 3 inputs: string protectedFilePath, string lockFilePath and int millisecondsToSleep

Reply:
I can change to take this parameter as an input but I was just wondering if its actually needed as it follows the same pattern across libraries

Reply:
Fair enough, unless @bgavrilMS has any objection. :-)

Now, how about we just have this new `cache_file_console_app` branch merge back to your `perf_testing` branch, and then we conclude this whole thing there?

Reply:
No, this is fine.